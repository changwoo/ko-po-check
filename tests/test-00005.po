# -*-text-*-
# test 5
#
msgid ""
msgstr ""
"Project-Id-Version: test 5\n"
"POT-Creation-Date: 2002-07-25 01:30+0900\n"
"PO-Revision-Date: 2002-07-25 01:30+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"

#: test.c:5
msgid "Some Properties"
msgstr "어쩌구 속성"

#: test.c:5
msgid "Properties : attribute"
msgstr "등록 정보: 속성"

#: test.c:5
msgid "Preferences"
msgstr "설정 사항"

#: test.c:5
msgid "Preferences"
msgstr "환경설정"

#: test.c:5
msgid "Preferences"
msgstr "기본 설정"

#: test.c:5
msgid "About..."
msgstr "...에 대하여"

#: test.c:5
msgid "About..."
msgstr "정보..."

#: test.c:5
msgid "_About"
msgstr "대하여(_A)"

#: test.c:5
msgid "Window font"
msgstr "창 폰트"

#: test.c:5
msgid "Window font"
msgstr "창 글꼴"

