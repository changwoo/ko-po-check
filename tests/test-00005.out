test-00005.po:17: 등록 정보: 그놈 데스크탑에서 "properties"은(는) "속성"(이)라고 번역
test-00005.po:25: 설정 사항: 그놈 데스크탑에서 "preferences"은(는) "기본 설정"(이)라고 번역
test-00005.po:29: 환경설정: 그놈 데스크탑에서 "preferences"은(는) "기본 설정"(이)라고 번역
test-00005.po:37: 대하여: 그놈 데스크탑에서 "about"은(는) "정보"(이)라고 번역
test-00005.po:45: 대하여: 그놈 데스크탑에서 "about"은(는) "정보"(이)라고 번역
test-00005.po:49: 폰트: 그놈 데스크탑에서 "font"은(는) "글꼴"(이)라고 번역
test-00005.po:58: 선택 사항: 그놈 데스크탑에서 "preferences"은(는) "기본 설정"(이)라고 번역
test-00005.po:68: 버튼: 그놈 데스크탑에서 "button"은(는) "단추"(이)라고 번역
test-00005.po:73: 삭제: 그놈 데스크탑에서 "delete"은(는) "지우기"(이)라고 번역
test-00005.po:78: 검색: 그놈 데스크탑에서 "find"은(는) "찾기"(이)라고 번역
test-00005.po:83: 검색: 그놈 데스크탑에서 "search"은(는) "찾기"(이)라고 번역
